engines/registry/app/models/access_token.rb

```diff
@@ -0,0 +1,46 @@
+require 'base32'
+
+# Following Docker distribution token auth specs,
+# see docs here: https://github.com/distribution/distribution/blob/main/docs/spec/auth/token.md
+class Registry::AccessToken
+  def initialize(account, service, granted_scopes)
+    @account = account
+    @service = service # "SUSE Linux OCI Registry"
```

Comment:
Implies a change on the registry, as it currently has `Docker`

Reply:
Not sure if any of the clients does care about this.

Reply:
Docker is a TM and a couple of years ago we had an effort to remove the word "Docker" from everything that was not referring directly to the command itself.